pom.xml

```diff
@@ -432,7 +433,7 @@
     <junit.version>4.13.1</junit.version>
     <testcontainers.version>1.15.2</testcontainers.version>
     <jedis.version>3.2.0</jedis.version>
-    <confluent.version>5.5.0</confluent.version>
+    <confluent.version>6.1.0</confluent.version>
```

Comment:
do you mind updating as well the version used for the integration test AlternativeKafkaContainer ?

Reply:
as well it would be interesting to do these changes in the github workflows, where it pass a 5.5 and a 6.1 version for the integration test.